note/doc/index.md

```diff
@@ -0,0 +1,83 @@
+```js script
+import { html } from 'lit';
+import '@divriots/dockit-core/note/dockit-note.define.js';
+```
+
+# dockit-note
+
+Component used to call out information in documentation pages.
+
+## Setup
+
+```js
+import '@divriots/dockit-core/note/dockit-note';
```

Comment:
nitpick: extension please, they are most important in code shown to users in docs, wherefore I keep an eye on it :)